build.gradle.kts

```diff
@@ -4,11 +4,33 @@ plugins {
 }
 
 configurations {
+  implementation { exclude(module = "spring-boot-starter-web") }
+  implementation { exclude(module = "spring-boot-starter-tomcat") }
   testImplementation { exclude(group = "org.junit.vintage") }
 }
 
 dependencies {
   implementation("org.springframework.boot:spring-boot-starter-webflux")
+  implementation("org.springframework.boot:spring-boot-starter-security")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+  implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+
+  implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
```

Comment:
Are we thinking to use coroutines and reactive for HTTP and DB calls?

Reply:
https://github.com/ministryofjustice/hmpps-non-associations-api/pull/3/commits/96180ba5dd4d3f67f1917ab2ce00642e9299cbca

Thoughts?

Reply:
All sounds good to me 😄

Reply:
Ah looks like we sent messages at the exact same time - https://github.com/ministryofjustice/hmpps-non-associations-api/pull/3#discussion_r1217735851 looks good